src/Chapter04/Listing04.20.ConstantInterpolatedStrings.cs

```diff
@@ -0,0 +1,21 @@
+#pragma warning disable CS0168 // Variable is declared but never used
+#pragma warning disable CS0219 // Variable is assigned but its value is never used
+#pragma warning disable IDE0059 // Unnecessary assignment of a value
```

Comment:
I feel like I keep seeing a lot of unnecessary suppressions, like here I don't think I see anything we don't use. And I think there's other cases of this as well that I'd look at.